Sources/X/Capability/Types/Genesis.pb.swift

```diff
@@ -0,0 +1,23 @@
+import CosmosProto
+
+// GenesisState defines the capability module's genesis state.
+struct CapabilityGenesisState: Codable {
+    // index is the capability global index.
+    let index: UInt64
```

Comment:
I think this guy may need to have the @StringBackedInt property wrapper if it is ever used as json.